sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityAdapter.java

```diff
@@ -15,19 +15,18 @@
  * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
  * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
  */
-final class SystematicSensitivityService {
-    private static final Logger LOGGER = LoggerFactory.getLogger(SystematicSensitivityService.class);
+final class SystematicSensitivityAdapter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SystematicSensitivityAdapter.class);
 
     static SystematicSensitivityResult runSensitivity(Network network,
-                                                               String workingStateId,
                                                                CnecSensitivityProvider cnecSensitivityProvider,
                                                                SensitivityAnalysisParameters sensitivityComputationParameters) {
-        LOGGER.debug("Sensitivity computation [start]");
-        SensitivityAnalysisResult result = SensitivityAnalysis.run(network, workingStateId, cnecSensitivityProvider, cnecSensitivityProvider, sensitivityComputationParameters);
-        LOGGER.debug("Sensitivity computation [end]");
+        LOGGER.debug("Sensitivity analysis [start]");
+        SensitivityAnalysisResult result = SensitivityAnalysis.run(network, network.getVariantManager().getWorkingVariantId(), cnecSensitivityProvider, cnecSensitivityProvider, sensitivityComputationParameters);
```

Comment:
This method is already defined in PowSyBl API you can remove the use of working variant ID

Reply:
done